lib/active_merchant/billing/gateways/realex.rb

```diff
@@ -184,6 +185,28 @@ def build_refund_request(money, authorization, options)
         xml.target!
       end
 
+      def build_credit_request(money, credit_card, options)
+        timestamp = new_timestamp
+        xml = Builder::XmlMarkup.new :indent => 2
+        xml.tag! 'request', 'timestamp' => timestamp, 'type' => 'credit' do
+          add_merchant_details(xml, options)
+          xml.tag! 'orderid', sanitize_order_id(options[:order_id])
+          add_amount(xml, money, options)
+          add_card(xml, credit_card)
+          xml.tag! 'refundhash', @options[:credit_hash] if @options[:credit_hash]
+          xml.tag! 'autosettle', 'flag' => 1
+          add_signed_digest(xml, timestamp, @options[:login], sanitize_order_id(options[:order_id]), amount(money), (options[:currency] || currency(money)), credit_card.number)
+          if credit_card.is_a?(NetworkTokenizationCreditCard)
+            add_network_tokenization_card(xml, credit_card)
+          else
+            add_three_d_secure(xml, options)
+          end
+          add_comments(xml, options)
+          add_address_and_customer_info(xml, options)
```

Comment:
I'm not sure whether I need to keep this line. It was part of the `purchase` request that I modeled this method after, but I don't know whether it's needed for `credit`.

Reply:
Same as above comment, this might not be needed, according to what I see in the docs.